webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPRuleRepository.scala

```diff
@@ -169,6 +169,44 @@ class WoLDAPRuleRepository(
     }
   }
 
+  private[this] def internalRuleSystemDirective(id:RuleId, modId: ModificationId, actor:EventActor, reason:Option[String]) : IOResult[DeleteRuleDiff] = {
```

Comment:
only this should be private and should have a Boolean to accept deleting system Rules

Reply:
ok i missread